src/zino/config/models.py

```diff
@@ -42,7 +42,7 @@ class PollDevice(BaseModel):
     retries: int = 3
     domain: str = None
     statistics: bool = True
-    hcounters: bool = False
+    hcounters: bool = True  # basically default to SNMP v2c
```

Comment:
Important comment. Worth documenting?

Reply:
Zino lacks proper documentation, which is a task of its own - so not sure where I would document this until a docs framework is in place.

I have suggested to Håvard that this option name should change.  It only makes sense when using Zino 1 for statistics collection.  High capacity counters are irrelevant for Zino 2, which doesn't provide a statistics collection replacement, so the option name makes no sense.

I.e. for now, the option name must remain the same in order to keep compatibility with old `polldevs.cf` files, but we should make an issue for replacing/renaming/aliasing it.

Reply:
#397